GreenGarbageApp/app/src/main/res/layout/fragment_review.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingTop="180dp"
+    tools:context="com.example.greengarbageapp.activities.MainActivity">
+
+    <TextView
```

Comment:
I would sugest to add a background, to make the text pop. 
android:background="@color/white"